radio/src/targets/horus/hal.h

```diff
@@ -934,4 +934,26 @@
   #define GPS_RCC_APB2Periph            0
 #endif
 
+//OW
+//MAVLINK_TELEM
+#if defined(RADIO_T16) && defined(PCBX10)
```

Comment:
`#if defined(TELEMETRY_MAVLINK)` as explained in the other comment

Reply:
done. thx.